frontend/src/components/editor/Editor.tsx

```diff
@@ -113,6 +286,14 @@ function Editor() {
 						minHeight: "100%",
 					}}
 				/>
+				{formatBarState.show && editorStore.cmView && (
+					<FormatBar
+						formatBarState={formatBarState}
+						setFormatBarState={setFormatBarState}
+						applyFormat={applyFormat}
```

Comment:
In CodePair, all function props in component starts with `on~~`.
ex) `onClick`, `onChange`, ...